### PR TITLE
Cumulative rev questions

### PR DESCRIPTION
**Pull request by nbuonin**

### CODE REVIEW

File: themes/ohcoe/layouts/cumulative-review/cumulative-review.html.html [hunk 1]
```diff
@@ -9,6 +9,9 @@
                 <h1>Cumulative Review Template</h1>
                 <h1>{{ .Title }}</h1>
                 {{ .Content }}
+                <div id="demographic-q-responses" style="display: none">
+                    <h1>Demograph Questions</h1>
```
Comment (sdreher) [inline themes/ohcoe/layouts/cumulative-review/cumulative-review.html.html]: Demographic?

File: themes/ohcoe/static/js/scripts.js [hunk 2]
```diff
@@ -151,4 +169,44 @@ $(function() {
...
+            }
+        };
+
+        var role = DEMO_ANSWERS.role[localStorage.getItem('role')];
+        var speciality = DEMO_ANSWERS.speciality[
+            localStorage.getItem('speciality')];
+
+        // If 'other' is set, and there's no value in the DEMO_ANSWERS
+        // for the given question, then just use the value in storage.
+        if (!role){
```
Comment (sdreher) [inline themes/ohcoe/static/js/scripts.js]: Some linting issues here.

File: themes/ohcoe/static/js/scripts.js [hunk 3]
```diff
@@ -151,4 +169,44 @@ $(function() {
     if (!localStorage.getItem('role') && !localStorage.getItem('speciality')) {
         $('#demographic-info').show();
     }
+
+    // Render on Cumulative Review page
+    if(document.getElementById('demographic-q-responses') &&
```
Comment (sdreher) [inline themes/ohcoe/static/js/scripts.js]: I'm seeing some bits here that could be a little clearer/tighter. For example, take a look at the number of times `localStorage.getItem('role')` is called here.

File: themes/ohcoe/static/js/scripts.js (hunk 2 above)
Comment (nbuonin) [reply themes/ohcoe/static/js/scripts.js]: What was the linting issue? It didn't show up in my editor or eslint.

File: themes/ohcoe/static/js/scripts.js (hunk 3 above)
Comment (nbuonin) [reply themes/ohcoe/static/js/scripts.js]: Got it, I've refactored this.

File: themes/ohcoe/layouts/cumulative-review/cumulative-review.html.html (hunk 1 above)
Comment (nbuonin) [reply themes/ohcoe/layouts/cumulative-review/cumulative-review.html.html]: Yep :(

File: themes/ohcoe/static/js/scripts.js (hunk 2 above)
Comment (sdreher) [reply themes/ohcoe/static/js/scripts.js]: oh weird, maybe we need to adjust the `.eslintrc` -- we usually follow this pattern:
`if (!role) {`

File: themes/ohcoe/static/js/scripts.js (hunk 3 above)
Comment (sdreher) [reply themes/ohcoe/static/js/scripts.js]: Nice, looks good.

File: themes/ohcoe/static/js/scripts.js (hunk 2 above)
Comment (nbuonin) [reply themes/ohcoe/static/js/scripts.js]: ach, ok I'll fix it

**Pull request merged**